### DISCUSSION TITLE
Local users are not available if LDAP enabled

### DISCUSSION THREAD

**Discussion by fabio-pif** [kimai/kimai#4451] [General]
After updating to version 2.4.0, with LDAP login enabled, local user (i.e. admin SUPER_USER created on installation via console) can't login.
To use local user, i had to set activate: false for ldap on local.yaml
Fabio

**Comment by kevinpapst** [comment]
Oh, that damn LDAP. I hate it ...

**Comment by fabio-pif** [comment]
😊 but is useful when you have already users in AD…

Let me know if I can help you somehow.
Thx
Fabio


Da: Kevin Papst ***@***.***>
Inviato: lunedì, 20 novembre 2023 15:02
A: kimai/kimai ***@***.***>
Cc: Fabio Pifferini ***@***.***>; Author ***@***.***>
Oggetto: Re: [kimai/kimai] Local users are not available if LDAP enabled (Discussion #4451)


Oh, that damn LDAP. I hate it ...

**Reply by kevinpapst** [reply]
Please comment here: https://github.com/kimai/kimai/issues/4452
